src/query/api/v1/handler/namespace/common.go

```diff
@@ -33,8 +34,22 @@ import (
 )
 
 const (
-	// M3DBNodeNamespacesKey is the KV key that holds namespaces
+	// M3DBServiceName is the service name for M3DB.
+	M3DBServiceName = "m3db"
+
+	// ServicesPathName is the services part of the API path.
+	ServicesPathName = "services"
```

Comment:
Nit: I'm not 100% sure these constants add much value; I think it may just make the URL's a bit harder to determine by quick inspection. Maybe worth just leaving them inlined?

Reply:
Yeah I was just following the pattern (that admitabbly I set up) in this file: https://github.com/m3db/m3/blob/master/src/query/api/v1/handler/placement/common.go

Reply:
Fair enough! I don't have too strong an opinion about it, so up to you.